.circleci/config.yml

```diff
@@ -106,10 +106,19 @@ jobs:
             sudo mkdir -p .tmp/attachments/staging
             sudo chmod -R 777 .
             cd support/integration-testing
-            npm run test:bruno
-            npm run test:bruno:clean
+            echo "Starting Bruno tests..."
+            timeout 1800s npm run test:bruno &
+            BRUNO_PID=$!
+            echo "Waiting for Bruno tests to complete..."
+            wait $BRUNO_PID
+            BRUNO_EXIT_CODE=$?
+            echo "Bruno tests finished with exit code: $BRUNO_EXIT_CODE"
+            echo "Cleaning up containers..."
+            timeout 120s npm run test:bruno:clean || (echo "Force stopping containers..." && docker compose -f ./docker-compose.bruno.yml down --timeout 30 --remove-orphans)
```

Comment:
I added this because CircleCI was having trouble stopping the containers itself using "abort on container exit" alone

Reply:
This is ok for now. We'll re-visit this.